packages/v4/src/content/design-guidelines/components/skeleton/skeleton.md

```diff
@@ -3,5 +3,17 @@ id: Skeleton
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+A **skeleton** is a type of loading state that shows an example of how data might be populated on a page before it gets populated. For content that may take a long time to load, use a [progress bar](/components/progress/react) in place of a skeleton.
+
+## Usage 
+Skeleton loading should be progressive - it should load static text first, followed by variable elements of the UI.
```

Comment:
I had to read this a few times to understand what it meant. Maybe there's a better way to say this? Perhap's we should say that skeleton loading allows you to expose content incrementally?

Reply:
Updated!